saleor/payment/gateways/braintree/plugin.py

```diff
@@ -32,6 +32,7 @@ def wrapped(self, *args, **kwargs):
 
 
 class BraintreeGatewayPlugin(BasePlugin):
+    PLUGIN_ID = "mirumee.gateway.braintree"
```

Comment:
A gateway can be many things, maybe "mirumee.payments.braintree" (and similarly for other payment plugins)?